tests/JsonErrorFactoryTest.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace WtfPhp\JsonApiErrors\Tests;
+
+use Exception;
+use PHPUnit\Framework\TestCase;
+use WtfPhp\JsonApiErrors\Exceptions\JsonApiErrorException;
+use WtfPhp\JsonApiErrors\Factories\JsonApiErrorFactory;
+use WtfPhp\JsonApiErrors\Models\JsonApiError;
+
+class JsonErrorFactoryTest extends TestCase
+{
+    /** @test */
+    public function itShouldReturnAJsonErrorObjectFromException()
```

Comment:
itShouldReturnA**JsonApiError**ObjectFromException

Could you also change the naming of the other functions?